packages/node/dist/index.js

```diff
@@ -1,4 +1,4 @@
-'use strict';
+import axios from 'axios';
```

Comment:
Why are you importing axios?

Reply:
I'm importing axios so i can use the mock framework to add test coverage to the bind function, so by using the axios in the bind method i make a get request to a static env.json so i can then bind it correctly to the window without requiring to write functionality to the index.html . the types changes are just to reflect this additional method being introduced and the removal of the default export, so does require to be together. possibly if we moved this pull request onto like a next so i can use it to get my feature out and then in a week or so i can do a small refactor to the cli to allow people to choose between js or json.